packages/server/src/app/user/user.router.js

```diff
@@ -158,16 +144,3 @@ userRouter.post(
     }
   }
 );
-
-userRouter.get('/:userId/challenges', jwtAuthHeaderValidator(), async (request, response, next) => {
```

Comment:
You're sure this route isn't used anymore? I'm getting a usage result when I look for the `/challenges` part of the end-point in the ChallengeService in the web client application?

Reply:
there exist one already on line 99

Reply:
Nevermind duplicate code removal is always good 🧹 🧹 🧹